index/index_model.go

```diff
@@ -60,6 +61,10 @@ type SpecIndexConfig struct {
 	// More details on relative references can be found in issue #73: https://github.com/pb33f/libopenapi/issues/73
 	BaseURL *url.URL // set the Base URL for resolving relative references if the spec is exploded.
 
+	// If resolving remotely, the RemoteDocumentGetter will be used to fetch the remote document.
+	// If not set, the default http client will be used.
+	RemoteDocumentGetter func(url string) (*http.Response, error)
```

Comment:
Add a link to the issue please, so others can follow the story.

Reply:
will do